cfme/tests/containers/test_labels.py

```diff
@@ -40,7 +40,10 @@
 def check_labels_in_ui(instance, name, expected_value):
     view = refresh_and_navigate(instance, 'Details')
     if view.entities.labels.is_displayed:
-        return view.entities.labels.get_text_of(name) == str(expected_value)
+        try:
+            return view.entities.labels.get_text_of(name) == str(expected_value)
+        except NameError:
```

Comment:
@mfalesni shouldnt those be lookuperrors from the wt side? - i just double checked and this one really irks me

Reply:
@RonnyPfannschmidt @mfalesni I added this exception to handle the case that we don't have this name in the table, I can do something like:
```python
name in view.entities.labels.read() and view.entities.labels.get_text_of(name) == str(expected_value)
```
instead.

Reply:
@RonnyPfannschmidt not sure, you tell me :)

Reply:
@gshefer I guess the question from Ronny was directed pretty much just towards me, so it does not apply on your PR.